src/Gulp/Tasks/Imagemin.ts

```diff
@@ -1,53 +1,40 @@
-import {
-    src,
-    dest,
-    parallel,
-    watch,
-    TaskFunction
-} from 'gulp';
-import debug from 'debug';
-import imagemin from 'gulp-imagemin';
-import { join } from 'path';
-import Project from '../../Models/Project';
-import Theme from '../../Models/Theme';
-import { TaskInterface } from './TaskInterface';
-const logger = debug('gpc:gulp:imageMin');
+import { src, dest, parallel, watch, TaskFunction } from "gulp";
+import debug from "debug";
+import squoosh from "gulp-libsquoosh";
+import { join } from "path";
+import Project from "../../Models/Project";
+import Theme from "../../Models/Theme";
+import { TaskInterface } from "./TaskInterface";
+const logger = debug("gpc:gulp:imageMin");
 import taskName from "./Decorators/TaskNameDecorator";
 
-@taskName("imageMin")
+@taskName("squoosh")
```

Comment:
Changed to `squoosh` from `imagemin` since `imagemin` will likely never be updated to support Apple Silicon.